.github/workflows/pull-request.yml

```
@@ -411,7 +424,7 @@
           echo "Starting Docker container for production..."
           docker compose -f docker/docker-compose.prod.yaml up -d
           echo "Production Docker container started successfully"
-
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)